processor/src/main/resources/org.mapstruct.ap.model.IterableMappingMethod.ftl

```diff
@@ -107,7 +107,7 @@
 <#macro iterableCreation>
     <@compress single_line=true>
         <#if factoryMethod??>
-            <@includeModel object=factoryMethod/>
+            <@includeModel object=factoryMethod targetType=resultType raw=true/>
```

Comment:
Adding ```raw=true``` is not necessary here and has no effect. Same for the MapMappingMethod. The only single place were we need to set ```raw=true``` is when adding ```<@includeModel ..>``` for a type to render. Actually, I have some clean-ups on that in PR #84.... If I had only known... :wink: